packages/app-mobile/utils/setUpNotifications.ts

```diff
@@ -0,0 +1,30 @@
+import { NativeEventEmitter, NativeModules, Platform } from "react-native";
```

Comment:
Please use `const { ... } = require('react-native')` when importing from React Native. For some reason, TypeScript sooner or later breaks on import/from statements. Not clear why because the error messages make no sense.

Reply:
Will do, this was automatic fix by VS Code. In general this file seems to fail multiple linter rules so I'll need to revisit.

I had to commit it with --no-verify because atm `root.tsx` is rejected by the linter (not because of my changes but because it's just too big) and I had both files in one commit.